ci/ci.sh

```diff
@@ -28,16 +28,17 @@ begingroup "Building Miri"
 export RUSTFLAGS="-D warnings"
 export CARGO_INCREMENTAL=0
 export CARGO_EXTRA_FLAGS="--locked"
+# We enable all features (except tracing) to make sure the Stacked Borrows consistency check runs.
+export FEATURES="--features=genmc,stack-cache,stack-cache-consistency-check"
```

Comment:
I am not happy about having to list the features here. I think my proposal would be that enabling the `tracing` feature just adds the corresponding code to the binary, but doesn't actually enable tracing. For that, we additionally have an env var, like `MIRI_TRACING` or so. Then one can also switch between traced and untraced builds more easily at runtime.